### ISSUE TITLE
Release models from the paper?

### ISSUE THREAD

**Issue by anikimmel** [QiujieDong/NeurCADRecon#7]
Is it possible for you to release models from the paper, as in the checkpoint files? To validate the results without retraining?

Thanks!

**Comment by QiujieDong** [comment]
Hi, @anikimmel,

NeurCADRecon is an overfitting method that does not require the dataset to retrain but only requires fitting a single model.

**Comment by anikimmel** [comment]
Yes, I am aware, but you reconstructed a ton of shapes and I was wondering if you could either release their siren models directly or the reconstructed shapes to validate your experiments without re-running all the training. 

Thanks!

**Comment by chl916185** [comment]
@QiujieDong 
@anikimmel 
What is the value of the overfitting method in practical applications, each data fits a checkpoint,  need a GT, but in a practical application, there is usually no GT.

**Comment by QiujieDong** [comment]
Hi, @chl916185,

Overfitting methods do not require GT.
